designdocs/passkey_p256.md

```diff
@@ -0,0 +1,101 @@
+# Design Doc for P256 PassKey Support
+
+## Table of Contents
+
+- [Design Doc for P256 PassKey Support](#design-doc-for-p256-passkey-support)
+  - [Table of Contents](#table-of-contents)
+  - [1. Introduction](#1-introduction)
+  - [2. Terminology](#2-terminology)
+    - [Accounts](#accounts)
+    - [Keys](#keys)
+    - [Signatures](#signatures)
+  - [3. Data Flow Diagram](#3-data-flow-diagram)
+  - [4. Data Map for Legal Teams](#4-data-map-for-legal-teams)
+  - [5. Specification](#5-specification)
+    - [Passkey Registration](#passkey-registration)
+    - [Security Considerations](#security-considerations)
+    - [Transaction Submission Specification](#transaction-submission-specification)
+  - [6. Implementation](#6-implementation)
+    - [Backend Example Code](#backend-example-code)
+    - [Frontend Example Code](#frontend-example-code)
+  - [7. Options for Discussion](#7-options-for-discussion)
+    - [Unsigned Extensions vs MultiSignature](#unsigned-extensions-vs-multisignature)
+    - [Generic Key Support](#generic-key-support)
+    - [Separate Pallet](#separate-pallet)
+  - [8. Conclusion](#8-conclusion)
+
+## 1. Introduction
+
+This document outlines the design considerations and specifications for integrating P256 Passkey support for performing transactions on Frequency chain. Passkey support aims to provide a novel non custodial solution for managing user accounts and signing transactions on-chain.
+
+## 2. Terminology
+
+### Accounts
+
+- **Authorized Account**: The user account associated with the Passkey.
+- **Parent Account**: The account holding the primary private key for transaction signing.
```

Comment:
What s the different from Authorized account and parent account? I think using `parent` is confusing in here since there no hierarchy like that.

Reply:
I renamed them to `Passkey Account` and `Frequency Account` where one is passkey and other is used for execution txs